public/img/logos/Adobe.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns='http://www.w3.org/2000/svg' x='0px' y='0px' width='100px' height='85.811px' viewBox='0 0 100 85.811' xml:space='preserve'><path fill-rule='evenodd' fill='#ED1C24' d='M49.608,32.625c0.104,0.105,23.395,53.232,23.373,53.186c-5.098,0-10.174,0-15.264,0 c0-0.164-7.23-17.32-7.155-17.41c-5.5,0-10.963,0-16.456,0C34.254,68.444,49.838,32.446,49.608,32.625L49.608,32.625z'/><path fill-rule='evenodd' fill='#ED1C24' d='M0,0c0.119,0,35.767,0,35.775,0C35.879-0.022,0.238,85.804,0.119,85.692 C0.119,85.572,0.038-0.06,0,0L0,0z'/><path fill-rule='evenodd' fill='#ED1C24' d='M63.009,0H100c0,0.216-0.024,85.856-0.024,85.736C99.997,86.014,63.009,0,63.009,0 L63.009,0z'/></svg>
```

Comment:
I didn't see in the ocean of diffs below, but we should also confirm that we're removing the old 3 logos in public/img/ (now that we're putting logos in a nested directory):

- Adobe.png
- AllMusic.png
- Linkedin.png